crates/timers/src/lib.rs

```diff
@@ -25,10 +25,13 @@ timeout.forget();
 ```
 
 ### Timeouts as `Future`s
-
-```no_run
+*/
```

Comment:
Can we add a note for users about the requirement for the feature being enabled right here?

Reply:
Sure thing! Thoughts on wording? First thing I came up with was this:

> With the `futures` feature enabled, a `future` module containing futures-based timers is exposed.